### ISSUE TITLE
[Design] Determine button line height

### ISSUE THREAD

**Issue by babsdenney** [department-of-veterans-affairs/vets-design-system-documentation#2984]
## Description
You can read the background about this ticket in the development ticket below.

[Update button line height #2963
](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2963)

We need to determine a new line height for buttons so that the button text doesn't run together. Once a new line-height is determined, update Figma and dev ticket with the new line height. 

## Details
[Button in Figma](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=199-1198)
[Button in Storybook](https://design.va.gov/storybook/?path=/docs/uswds-va-button--primary)

## Tasks

- [ ] Determine the button line height 
- [ ] Review with DST Designers, PO, and PM
- [ ] Review designs with developers and accessibility specialist
- [ ] Update all Figma components with the new line height
- [ ] Update [this ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2963) with the line height needed to be implemented in code
- [ ] Address any comments from reviews, if necessary
- [ ] Comment on this ticket with any accessibility considerations engineers may need to know
- [ ] Comment on this ticket with content specifications (e.g. labels and error messages)
- [ ] Comment on this ticket with a link to the designs and post in DST Slack channel

## Acceptance Criteria
- [ ] Component design is complete and has been reviewed
- [ ] Accessibility and development considerations have been added to this ticket, if necessary
- [ ] Content specifications have been added to this ticket, if necessary
- [ ] Link to design has been added to this ticket and shared in Slack

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2984?planning-poker) with Zenhub @babsdenney @danbrady @LWWright7

**Comment by LWWright7** [comment]
@danbrady @babsdenney - This is ready for [review](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=199-1198&t=YGwgph2zwPt5Ah1M-1).  I used the USWDS spacing for guidance and created examples to show what the button would look like with spacing at the USWDS breakpoints (1, 1.15, 1.35, 1.5, 1.62, 1.75 and I added 2 just for funzies).

**Comment by LWWright7** [comment]
This ticket is blocked by a research ticket for dev: [3088](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3088)

**Comment by LWWright7** [comment]
Closing this ticket as the line height has been determined through research into USWDS breakpoints in this ticket:

![Screen Shot 2024-07-25 at 10 32 27 AM](https://github.com/user-attachments/assets/268876db-f6a1-4003-8dfa-5096c6ef4f01)

I've created a [new ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3089) to actually update Figma component depending on the research that the dev team does in [this ticket. ](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3088)
